### ISSUE TITLE
Overfitting on validation data?

### ISSUE THREAD

**Issue by saidineshpola** [open-mmlab/mmdetection#7481]
Hi,
when I evaluate my trained HTC model on validation images  I got 20.2 mAP but when I evaluate the same model on test images I got 15mAP.  I tried all of the models, from 9 epochs to 14 epochs, and on test photos, I got about 15mAP. And there is no data leakage between train and val images. I am confused why  mask segmentation on test images is not improving but it is on validation images even though both have same data pipeline. Should I change test_cfg in model config?  I don't know what to do here . For your information the train dataset has 50k images while test dataset has 3k images. Can anybody assist me? thanks.

**Comment by Czm369** [comment]
Excuse me, which dataset  are you using?

**Comment by saidineshpola** [comment]
The dataset is from aicrowd's food recognition challenge
